pyvista/plotting/prop3d.py

```diff
@@ -0,0 +1,223 @@
+"""Prop3D module."""
+from pyvista import _vtk
+
+
+class Prop3D(_vtk.vtkProp3D):
+    """Prop3D wrapper for vtkProp3D.
+
+    Used to represent an entity in a rendering scene. It handles functions
+    related to the position, orientation and scaling. Used as parent class
+    in Actor and Volume class.
+    """
+
+    def __init__(self):
+        """Initialize Prop3D."""
+        super().__init__()
+
+    @property
+    def scale(self) -> tuple:
```

Comment:
I'd like to make this return type a bit more explicit. e.g. `Tuple[Number, Number, Number]`

Reply:
Or perhaps `Vector` from `pyvista._typing`?

Reply:
Going to go with just `Tuple[float, float, float]` since scale doesn't really feel like a `Vector` to me in the sense of a scale factor.

Reply:
Actually, returning a `Tuple[Number, Number, Number]` but accepting a `Vector`. We always return a tuple and accept all types in `Vector`.